### DISCUSSION TITLE
Shortcuts not working

### DISCUSSION THREAD

**Discussion by iandk** [spatie/ray#789] [General]
Hi,

I've been having issues with Ray shortcuts since several months.
It worked just fine in the beginning but then stopped working at some point.

I tried several different shortcut combinations, but none of them work.
I'm using the latest version of ray on macOS 13.0.1.

Reinstalling did not help.

**Comment by freekmurze** [comment]
Which shortcut is not working for you? They all seem to work on my end. 
Maybe there's another app conflicting? You can change the shortcuts in the preferences.

**Comment by iandk** [comment]
It affects all of them.
Neither clearing nor hiding ray works.


I tried several different combinations, none of them work.
Ray just stopped working at some point, no idea why
